pkg/dynamicrp/config.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dynamicrp
+
+import (
+	"bytes"
+
+	"github.com/radius-project/radius/pkg/armrpc/hostoptions"
+	metricsprovider "github.com/radius-project/radius/pkg/metrics/provider"
+	profilerprovider "github.com/radius-project/radius/pkg/profiler/provider"
+	"github.com/radius-project/radius/pkg/trace"
+	ucpconfig "github.com/radius-project/radius/pkg/ucp/config"
+	"github.com/radius-project/radius/pkg/ucp/dataprovider"
+	queueprovider "github.com/radius-project/radius/pkg/ucp/queue/provider"
+	secretprovider "github.com/radius-project/radius/pkg/ucp/secret/provider"
+	"github.com/radius-project/radius/pkg/ucp/ucplog"
+	"gopkg.in/yaml.v3"
```

Comment:
nit: extra line before this import for the grouping of imports

Reply:
If gofmt is happy with it, then so am I 😆